docs/IE_DG/ONNX_Support.md

```diff
@@ -9,10 +9,41 @@ InferenceEngine::Core core;
 auto network = core.ReadNetwork("model.onnx");
 ```
 
+**Reshape feature:**
+
 OpenVINO™ doesn't provide a mechanism to specify pre-processing (like mean values subtraction, reverse input channels) for the ONNX format.
 If an ONNX model contains dynamic shapes for input, please use the `CNNNetwork::reshape` method for shape specialization.
 
-Unsupported types of tensors:
+**Weights saved in external files:**
+
+OpenVINO™ supports ONNX models which use weights saved in external files. It is especially useful for models larger than 2GB because of protobuf limitations.
+If you want to read such model you should use `ReadNetwork` overload which takes `modelPath` as input parameter (both `std::string` and `std::wstring`).
+Note that `binPath` argument of `ReadNetwork` should be empty in such case, because paths to external weights are saved directly in an ONNX model.
```

Comment:
```suggestion
Note that the `binPath` argument of `ReadNetwork` should be empty in this case, because paths to external weights are saved directly in an ONNX model.
```